setup.py

```diff
@@ -38,7 +32,8 @@
 setup(
     name='iotedgedev',
     version='3.1.2',
```

Comment:
Should I pump the version?

Reply:
Yes, let's update the version. I wonder if we should work in a feature branch until we are ready with all changes we want to publish. What do you think? Or should we publish each use case under a separate version?

Reply:
Let's update the version when we are ready to release. For now, keep it as it is.